e2e/load_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"bytes"
+	"log"
+	"testing"
+	"time"
+
+	"fortio.org/fortio/fhttp"
+	"fortio.org/fortio/fnet"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_http_load_with_high_bandwidth(t *testing.T) {
+	stdErr, kill := startEnvoyWith("network", t, 8001)
```

Comment:
Network filter is not representative(it’s just logging the body and that’s it) so could you please write some more realistic example extension for load testing? ( e.g. header manipulation+AuthZ, etc)

Reply:
done

Reply:
Not done.